native/src/components/FeedbackContainer.tsx

```diff
@@ -22,9 +22,17 @@ export type FeedbackContainerProps = {
   cityCode: string
   query?: string
   slug?: string
+  noResults: boolean
```

Comment:
```suggestion
  noResults?: boolean
```
I think there is noResults only when I search and 0 results so it's false most of the time until we set it to true.